test/index.js

```diff
@@ -808,4 +814,430 @@ describe('function helpers', () => {
       'Failed to download https://example.com/nonexistentfile: 404 Not Found',
     )
   })
+
+  describe('config', () => {
```

Comment:
[pebble] something to address in future work but I think we need to start breaking this test file down into different files

Reply:
Yeah, that occurred to me as well, but I thought I'd wait after we wrap up all our convos we've been having about testing in general.